rtfm/source/_pages/usage/css_js.rst

```diff
@@ -0,0 +1,76 @@
+.. _Assets:
+
+SCSS and JS
+===========
+
+The Fast Way
+""""""""""""
+
+- write your SCSS in ``src/stylesheets/screen.scss``
+- write your JS in ``src/javascripts/application.js``
+
+and all will automagically work! :)
+
+A real explanation
+""""""""""""""""""
+
+Wordless has 2 different places where you want to put your assets (javascript,
+css, images):
+
+- Place all your custom, project related assets into ``src/*``
+- Since you are backed by Webpack, you can use NPM (``node_modules``) to import new dependencies
+  following a completely standard approach
+
+Custom assets
+^^^^^^^^^^^^^
+
+They must be placed inside ``src/javascript/`` and
+``src/stylesheets/`` and ``src/images/``.
+
+They will be compiled and resulting compilation files will be moved in the corresponding
+``assets/xxx`` folder.
+
+Compilation, naming and other logic is fully handled by webpack.
+
+Images will be optimized by `ImageminPlugin`_. The default setup already translates
+``url`` s inside css/scss files in order to point to images in the
+right folder.
+
+.. _ImageminPlugin: https://www.npmjs.com/package/imagemin-webpack-plugin
```

Comment:
as before the new link is https://github.com/webpack-contrib/image-minimizer-webpack-plugin